mne/beamformer/tests/test_dics.py

```diff
@@ -44,7 +44,7 @@ def _get_data(tmin=-0.11, tmax=0.15, read_all_forward=True, compute_csds=True):
     if read_all_forward:
         forward_surf_ori = _read_forward_solution_meg(fname_fwd, surf_ori=True)
         forward_fixed = _read_forward_solution_meg(fname_fwd, force_fixed=True,
-                                                   surf_ori=True)
+                                                   use_cps=False)
```

Comment:
it's not only the dics test that would fail.  There are basically 3 failing tests if I allow only use_cps=True: 
- test_dics.py (beamformer module)
- test_inverse.py (minimum norm module)
- test_evoked.py (simulation module)

For example, in test_inverse.py for minimum norm, there is a test, which explicitely requires to generate a fixed forward operator without cps.

Do we want to dismiss the option to generate a fixed fwd without cps?